samples/authorization/csm_authorization_proxy_server_v1100.yaml

```diff
@@ -37,6 +37,7 @@ spec:
         tenantService: dellemc/csm-authorization-tenant:v1.10.0
         roleService: dellemc/csm-authorization-role:v1.10.0
         storageService: dellemc/csm-authorization-storage:v1.10.0
+        vaultAddress: https://10.0.0.1:8400
```

Comment:
Can these changes be added to the `v2.0.0-alpha` samples file?

Reply:
Moved to v2.0.0-alpha